### DISCUSSION TITLE
How to log a terminal session with zsh-autocomplete installed?

### DISCUSSION THREAD

**Discussion by dazu89** [marlonrichert/zsh-autocomplete#710] [Q&A]
My gratitude to @marlonrichert for making zsh-autocomplete available, I am an avid user. However, today I had the need to log my current terminal session, using  `script terminal.log`, however, each keystroke is recorded which makes the file barely readable. So I wondered, what zsh-autocomplete users might resort to in order to obtain a clean and readable terminal log.
Thank you for a comment.

**Comment by marlonrichert** [comment]
I think you'll just have to do it without Autocomplete. 🙂
